common/scala/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/CosmosDBArtifactStore.scala

```diff
@@ -240,10 +252,17 @@ class CosmosDBArtifactStore[DocumentAbstraction <: DocumentSerializer](protected
       .map(_.toList)
       .map(l => if (limit > 0) l.take(limit) else l)
 
-    f.foreach { out =>
-      transid.finished(this, start, s"[QUERY] '$collName' completed: matched ${out.size}")
+    val g = f.andThen {
+      case Success(out) =>
+        queryMetrics.foreach { m =>
+          val combinedMetrics = QueryMetrics.ZERO.add(m: _*)
+          logging.debug(
+            this,
+            s"[QueryMetricsEnabled] Collection [$collName] - Query [${querySpec.getQueryText}].\nQueryMetrics\n[$combinedMetrics]")
```

Comment:
what shows up in `$combinedMetrics` - do you have some sample output? I don't see it in the test

Reply:
The sample output is there in PR description. The testcase currently only looks for a marker `QueryMetricsEnabled` to assert that log statement is executed